zeppelin-web/src/app/notebook/paragraph/result/result.controller.js

```diff
@@ -755,7 +755,7 @@ function ResultCtrl ($scope, $rootScope, $route, $window, $routeParams, $locatio
     if (paragraphId === paragraph.id) {
       let builtInViz = builtInVisualizations[$scope.graphMode]
       if (builtInViz && builtInViz.instance) {
-        builtInViz.instance.resize()
+        $timeout(_ => builtInViz.instance.resize(), 200)
```

Comment:
why this?

Reply:
Column size change needs some time to reflect on DOM